packages/create-pantheon-decoupled-kit/src/templates/next-drupal/components/layout.jsx.ts

```diff
@@ -0,0 +1,10 @@
+import { TemplateFn, isDrupalCms } from '@cli/types';
+import { layoutTemplate } from '@partials/nextjs-shared/layout';
+
+const jsx: TemplateFn = ({ data }) =>
+	/* jsx */ `${layoutTemplate({
+		search: data.search,
+		cmsType: isDrupalCms(data.cmsType) ? 'drupal' : 'false',
```

Comment:
I think it's fine to keep this as `data.cmsType`, but you can use the type guard in the `layoutTemplate`
```suggestion
		cmsType: data.cmsType,
```